libraries/importers/src/main/java/org/eclipse/sw360/importer/ComponentCSVRecord.java

```diff
@@ -717,7 +717,7 @@ public String getVendorName() {
         elements.add(SampleOptions.URL_OPTION);
         elements.add(SampleOptions.URL_OPTION);
         elements.add(SampleOptions.URL_OPTION);
-
+        
```

Comment:
This is no review, just by scrolling over the changeset these whitespaces in blank lines were something that I stumbled across. They seem to be in some java files.

Reply:
It was a feedback from nutan that these files didn't have relevant changes therefore it would be better to revert them.
Since this is a new PR now, it might be better to remove them from this change as a revert now does not have any advantages.